src/content-handlers/iiif/extensions/uv-aleph-extension/config/config.json

```
@@ -110,6 +110,7 @@
         "feedback": "Feedback",
         "fullScreen": "Full Screen",
         "moreInfo": "More Information",
+        "toggleLeftPanel": "Toggle Left Panel",
```

I think this should be using the shorthand versions of everything for proper translation.

```suggestion
        "toggleLeftPanel": "$toggleLeftPanel",
```